docs/src/installation.md

```diff
@@ -157,6 +157,11 @@ Using [MacPorts](https://www.macports.org):
 ```sh
 sudo port install sbt
 ```
+or with [Homebrew](https://brew.sh/)
```

Comment:
```suggestion

Using [Homebrew](https://brew.sh/):
```

Stylistic nit.

Reply:
I am using the Homebrew version of sbt. Just installed it and reinstalled it to check. The issue is that brew then installs Java 21, which is broken with sbt (similar to the issue you link to). After uninstalling Java 21 from brew (`brew uninstall --ignore-dependencies java`) it works again. So I don't know what is best. As many MacOs users are using brew, I would include it. Maybe add the warning about Java 21.